src/common/Web3Client.ts

```diff
@@ -82,7 +84,7 @@ export default class Web3Client {
     // @todo handle hex values of gas
 
     if (web3Options.parent) {
-      _web3Options.gas = (_web3Options.gas || this.parentDefaultOptions.gas) + 1000000
+      _web3Options.gas = (_web3Options.gas || this.parentDefaultOptions.gas) + ONEMIL
```

Comment:
Why 1 million gas is being added here? Give this constant more sound name + add short comment what is the reason to increase the gas that was configured by the user.